shared/naturalcrit/markdown.js

```diff
@@ -206,6 +206,38 @@ const mustacheInjectBlock = {
 	}
 };
 
+const superSubScripts = {
+	name  : 'superSubScripts',
+	level : 'inline',
+	start(src) { return src.match(/\^\^[^\s].+[^\s]\^\^/m)?.index; },  // Hint to Marked.js to stop and check for a match
```

Comment:
This start script still shouldn't have a `.+` in it because that means if it sees a `^^` it will scan the entire document including newlines for the largest possible string to fit inside the closing `^^`. This will become very slow. We want to limit this search to one line.

Actually, for Marked.js, this "start" parameter can just be the first few characters. We might simplify this to just this:
```suggestion
	start(src) { return src.match(/\^\^/m)?.index; },  // Hint to Marked.js to stop and check for a match
```